### PR TITLE
Add an article about jackdaw.admin

### PR DESCRIPTION
**Pull request by AlexVPopov**
This is a first stab at providing a small guide to the `jackdaw.admin` namespace.

# Checklist

- ~[] tests~
- ~[ ] updated [CHANGELOG](../CHANGELOG.md) (the "unreleased" section)~

### CODE REVIEW

File: doc/admin.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### Viewing topics
+
+To simply list the names of the topics in the cluster, use the `list-topics` function:
+
+```clojure
+(ja/list-topics client) ;; => ({:topic-name "jackdaw"})
+```
+
+To view the parition information for a topic, use `describe-topics` function:
```
Comment (cddr) [inline doc/admin.md]: s/parition/partition/

File: doc/admin.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+;;                                                                        :name "cleanup.policy",
+;;                                                                        :read-only? false,
+;;                                                                        :sensitive? false,
+;;                                                                        :value "compact"},
+;;                                                                                     ...}}
+```
+
+### Changing topics' configuration
+
+To change the configuration of a topic, pass the function `alter-topic-config!` a client and an array of topic descriptor maps. Each map needs to contain two keys - the `:topic-name`, as well as a `:topic-config`, holding the configuration<sup>[3](#topic-configs)</sup> of the topic as stringified key-value pairs.
```
Comment (cddr) [inline doc/admin.md]: Maybe consider switching the "array" in this line for "vector"?

Comment (AlexVPopov) [reply doc/admin.md]: Absolutely! Coming from Ruby I always tend to say `array` and `hash`, instead of `vector` and `map`, well spotted.

**Comment by AlexVPopov** [comment]
@cddr PR updated.

**Pull request merged**